src/Runner.js

```diff
@@ -168,27 +179,28 @@ export default class Runner extends EventEmitter {
     }
     if (process.platform === 'win32') {
       // Windows doesn't exit the process when it should.
-      spawn('taskkill', ['/pid', '' + this.debugprocess.pid, '/T', '/F']);
+      spawn('taskkill', ['/pid', `${this.debugprocess.pid}`, '/T', '/F']);
```

Comment:
Does the code formatter really suggest using a template literal `` `${arg}` `` over the explicit type coercion `String(arg)`? You ES-Next kids are so fancy! 🚀  https://eslint.org/docs/rules/no-implicit-coercion

Reply:
this is most likely determined by the eslint, in `.eslintrc.js`:
```
extends: ['airbnb-base', 'plugin:flowtype/recommended', 'prettier', 'plugin:prettier/recommended'],
```
I don't have any preference actually, it seems fine to me either way as long as we are consistent in the code base.